.github/workflows/r-ci.yml

```diff
@@ -112,17 +101,22 @@ jobs:
           Rscript -e 'print(packageVersion("tiledb"))'
           Rscript -e 'print(tiledb::tiledb_version())'
 
-      - name: Test
-        if: ${{ matrix.covr == 'no' }}
+      - name: We want tiledb-r 0.21 and core 2.17
         run: |
           cd apis/r
+          Rscript -e 'remove.packages("tiledb")'
+          wget https://github.com/TileDB-Inc/TileDB-R/archive/refs/tags/0.21.3.tar.gz
+          Rscript -e 'remotes::install_deps("0.21.3.tar.gz")'
+          R CMD INSTALL 0.21.3.tar.gz    # as 0.21.3 is needed by SOMA 1.5.1
```

Comment:
That is doing work twice.  You just installed it.

Reply:
@eddelbuettel I am iterating until I get green CI with 2.17 and 0.22 surviving all the way until the test runs. I'm not there yet. When I finally succeed, I'll get more parsimonious. Thank you!

Reply:
@johnkerl if you install the same package twice you are waiting longer.  Is that really on purpose?

Reply:
@eddelbuettel again, I'm running experiments, iterating, trying to find out where the downgrade is happening ...

Reply:
@johnkerl I understand I am just explaining that your wait is longer if there is a step you need _once_ but do _twice_.  Above you install the same package twice.  But whatevs.

Reply:
@eddelbuettel thanks

CI is now green, and I will now prune